tests/integration/cmd_debug_test.go

```diff
@@ -38,6 +38,27 @@ var _ = Describe("odo debug command tests", func() {
 	})
 
 	Context("odo debug on a nodejs:latest component", func() {
+		It("check that machine output debug information works", func() {
+			helper.CopyExample(filepath.Join("source", "nodejs"), context)
+			helper.CmdShouldPass("odo", "component", "create", "nodejs:latest", "--project", project, "--context", context)
+			helper.CmdShouldPass("odo", "push", "--context", context)
+
+			stopChannel := make(chan bool)
+			go func() {
+				helper.CmdShouldRunAndTerminate(60*time.Second, stopChannel, "odo", "debug", "port-forward", "--local-port", "9393", "--context", context)
```

Comment:
Please use the util function HttpGetFreePort() for finding the free port and use it instead of `9392. It might lead to flake in the future.
https://github.com/openshift/odo/pull/2653/files#diff-3177c5007157ee638470e90363d3cf6dR128

Reply:
Thanks man! Updated.